main.py

```diff
@@ -2,6 +2,29 @@
 
 from django.db.models import QuerySet
 
+from db.models import Genre, Actor
+
 
 def main() -> QuerySet:
-    pass
+    genres = ["Western", "Action", "Dramma"]
+    actors = [("George", "Klooney"),
```

Comment:
There is a typo in the actor's last name 'Klooney'. It should be 'Clooney' to match the intended correction later in the code.